packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -843,6 +843,7 @@ const ruleset$1 = {
             description: "Operation should have a summary or description.",
             message: "Operation should have a summary or description.",
             severity: "warn",
+            ignoreForTypespec: true,
```

Comment:
Ideally each such line should have a comment explaining why this is here. Either provide a URL showing where is the validation that replaces this LintDiff validation, or provide a URL to the issue to add such validation (and ideally that issue would say why it is OK to disable this).

If it is not possible to add comments to these ruleset files (not sure if it is) then maybe we could add extra ignored field like `ignoreForTypespecReason`.

Reply:
I would also feel more confident if we could add tests for each rule, where we take a small bit of typespec, generate swagger, then run both tools and make sure the typespec rule covers.